sky/shell/platform/ios/sky_dynamic_service_loader.h

```diff
@@ -0,0 +1,18 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef SKY_SHELL_PLATFORM_IOS_SKY_DYNAMIC_SERVICE_LOADER_H_
+#define SKY_SHELL_PLATFORM_IOS_SKY_DYNAMIC_SERVICE_LOADER_H_
+
+#import <Foundation/Foundation.h>
+#include "sky/shell/platform/mac/platform_service_provider.h"
+
+@interface SkyDynamicServiceLoader : NSObject
```

Comment:
I'm little sad about the name "sky" in the ABI.  We should probably do a pass soon where we remove all the "sky" and "domokit" names before it's too late.